README.md

```diff
@@ -4,3 +4,18 @@ OnlineSnake.fun is a small fun multiplayer browser game.
 
 ### Background
 This project was started when I was volunteering at the [Junction 2018](https://hackjunction.com/). I wanted to try out real-time web programming with websockets and decided to make a simple game. This is the result.
+
+### Development setup
+#### Docker container
+In project repository, run `docker-compose up`. After the container is ready, navigate to [localhost:8080](http://localhost:8080) using your web browser. If you make changes to web files (client), refresh the web page to see the effect. If you change server files (python), restart the docker container to see changes.
+
+#### Without Docker
+In project repository:
+```bash
+virtualenv -p python3 venv
```

Comment:
Should we encourage the use of the Python [venv](https://docs.python.org/3/library/venv.html) module like this?
```suggestion
python3 -m venv venv
```
[virtualenv](https://virtualenv.pypa.io/en/latest/) has more features but venv module is shipped with Python

Reply:
At least on my Ubuntu system the venv module is not installed with python3 :thinking: 
```
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.
    apt-get install python3-venv
```
But I guess venv module would be actually better in this case, as it belongs to standard library (while virtualenv is a third party package) and supports the features that are needed in this project.

Additionally, this link [https://docs.python.org/3/library/venv.html](https://docs.python.org/3/library/venv.html) could be added as it includes instructions for other shells than zsh/bash.